### PR TITLE
SOCINT 105 Update CSP

### PR DESCRIPTION
**Pull request by Paul-Joel**
# Motivation and Context
Make the AIMS URL declared in the CSP dynamic so it works for all environments via the environment variable value

# What has changed
Update to build of CSP values to add in aims url dynamic from config values, including unit test updates
Additional of docker-compose for redis (previously used ras-rm version no longer available), and update of redis port number in config
Removal of .travis.yml to stop travis build on push

# How to test?
When deployed, the RHUI typeahead implementation should now work whatever the environment, and not violate the CSP rules
Travis should no longer build on push.

Changes tested on sdc-rh-dev

# Links
https://collaborate2.ons.gov.uk/jira/browse/SOCINT-105

### CODE REVIEW

File: app/security.py [hunk 1]
```diff
@@ -78,6 +78,10 @@ def _format_csp(csp_dict):
     'script-src',
```
Comment (rpcatling) [inline app/security.py:78]: line 68 could be deleted, and also `_format_csp` method I think, or is it for some debug purposes? in which case add a comment to say so

File: docker/docker-compose-redis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#
+# This file is a placeholder for a docker compose file that will start the rh-ui.
```
Comment (rpcatling) [inline docker/docker-compose-redis.yml:2]: This should be mentioned in the README for part of the setup for running locally.

**Review by rpcatling — CHANGES_REQUESTED** [review]
I think the opportunity to update the README, including removing Census specific stuff should be taken.
The instructions for running locally should include the new docker-compose for redis among other things.

File: app/security.py (hunk 1 above)
Comment (Paul-Joel) [reply app/security.py:70]: Removed code lines - hangover from a long ago change that isn't used any more

File: docker/docker-compose-redis.yml (hunk 2 above)
Comment (Paul-Joel) [reply docker/docker-compose-redis.yml:2]: This is implemented and documented properly in Peter's changes - I'll do a proper review of the Readme when both changes are merged

Comment (rpcatling) [reply docker/docker-compose-redis.yml:2]: ok sounds fine

**Review by rpcatling — APPROVED** [review]

**Review by philwhiles — APPROVED** [review]

**Pull request merged**